src/main/java/core/basesyntax/ColorList.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum ColorList {
+    red,
+    blue,
+    yellow,
+    green,
+    white,
+    black,
+    orange
```

Comment:
let's write them in upper case
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names